pandas/core/dtypes/missing.py

```diff
@@ -297,6 +250,24 @@ def _isna_ndarraylike_old(obj):
     return result
 
 
+def _isna_string_dtype(values: np.ndarray, dtype: np.dtype, old: bool) -> np.ndarray:
+    # Working around NumPy ticket 1542
```

Comment:
not even sure this is the correct issue anymore :-< but can revist this whole thing later